src/components/Form.js

```diff
@@ -23,10 +23,25 @@ const StyledInner = styled.div`
   background: ${brandColours.primary};
   border-radius: 10px;
   width: 100%;
+  padding: 20px;
+
+  ${minBreakpointQuery.small`
+    padding-top: 40px;
```

Comment:
If your changing them all could just do this shorthand, padding: 40px 50px;, but it if was just a couple then ye individual is better.